front/.eslintrc.js

```diff
@@ -166,16 +165,6 @@ module.exports = {
         markers: ['/'],
       },
     ],
-    '@typescript-eslint/tslint/config': [
```

Comment:
This is the only tslint custom rule we are still using. It is preventing you to directly import code from the modules into the main app. Before we can delete tslint (which I agree we should) this rule needs to be replaced by an eslint one to make sure we keep these imports in check.